packages/react-native-dogfood/src/screens/Meeting/GuestMeetingScreen.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { useEffect, useState } from 'react';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import {
+  StreamCall,
+  StreamVideo,
+  TokenOrProvider,
+  UserResponse,
+  useCall,
+  useCreateStreamVideoClient,
+} from '@stream-io/video-react-native-sdk';
+import { MeetingStackParamList, ScreenTypes } from '../../../types';
+import {
+  startForegroundService,
+  stopForegroundService,
+} from '../../modules/push/android';
+import { MeetingUI } from '../../components/MeetingUI';
+import { createToken } from '../../modules/helpers/jwt';
+
+type Props = NativeStackScreenProps<
+  MeetingStackParamList,
+  'GuestMeetingScreen'
+>;
+
+export const GuestMeetingScreen = (props: Props) => {
+  const apiKey = process.env.STREAM_API_KEY as string;
+  const apiSecret = process.env.STREAM_API_SECRET as string;
+  const {
+    params: { guestUserId, guestCallId, mode },
+  } = props.route;
+  const guestCallType = 'default';
+
+  const user: UserResponse = {
+    id: `anonymous-${Math.random().toString(36).substring(2, 15)}`,
+    created_at: new Date().toISOString(),
+    updated_at: new Date().toISOString(),
+    role: 'guest',
+    teams: [],
+    custom: {},
+  };
```

Comment:
better to move it inside useState().. having it outside seems unuseful

Reply:
I feel this is clear

Reply:
imagine someone looking at this code a month after.. they might be wondering why is user created on every render.. but only used at the initial state

Reply:
I would have suggested moving outside of the render function... but date needs to created newly at mount, that cannot be done